src/app/series/directives/series-feeds.component.ts

```diff
@@ -41,6 +84,11 @@ export class SeriesFeedsComponent implements OnDestroy {
         this.loading = false;
       }
     });
+    if (series.doc.has('prx:tags')) {
+      series.doc.follow('prx:tags').subscribe((tags) => {
+        this.includeTagOptions = (tags['tags'] || []).map((t) => [t, t]);
```

Comment:
One bit of oddness:

Because this CMS endpoint does a mysql `SELECT DISTINCT`, you end up with a list of _case insensitive_ unique values here.  So if you tagged one episode with `hello` and another with `HeLLo` ... you'd only see 1 of those 2 values here.

BUT - as the Feeder logic for `include_tags` already does a bunch of normalization, things will work out as expected here.  And in fact, Feeder [goes further](https://github.com/PRX/feeder.prx.org/blob/master/app/models/feed.rb#L92) to normalize them, restricting characters.  So you might end up matching episodes you didn't expect.  Though in reality, I'll bet that doesn't happen.